code/modules/roguetown/roguemachine/questing/monsters/questbehemoth.dm

```diff
@@ -0,0 +1,163 @@
+/mob/living/simple_animal/hostile/retaliate/rogue/elemental/questbehemoth
```

Comment:
Could save the headache and make it a subtype, if it's identical for most of what's here.